packages/suite/src/hooks/wallet/form/useCompose.ts

```diff
@@ -37,6 +41,7 @@ export const useCompose = <TFieldValues extends FormState>({
     getValues,
     formState: { errors },
     clearErrors,
+    txStakeName,
```

Comment:
I am not fan of this, I will take a look how to improve it.

Reply:
Me neither to be honest. Just didn't find a way how to do it in a better way. Waiting for your suggestions.